modules/lib/src/main/resources/assets/js/app/inputtype/ui/text/dialog/LinkModalDialog.ts

```diff
@@ -55,12 +64,15 @@ export class LinkModalDialog
     private textFormItem: FormItem;
     private toolTipFormItem: FormItem;
     private mediaOptionRadioFormItem: FormItem;
+    private protocolsDropdownButton: MenuButton;
 
     private contentId: ContentId;
     private parentSitePath: string;
 
     private tabNames: any;
 
+    private urlProtocols: UrlProtocol[];
```

Comment:
This property will never change, so it can be defined as `static readonly` and be defined with default value straight away.